tests/bad_udta_trailer.mov.txt

```diff
@@ -0,0 +1,4 @@
+[udta] UserData @ 0x0, 0x8 .. 0xc (4)
+    ERROR: Optional udta trailer must be a single 32-bit zero, found: b'XXXX'
+
+    58 58 58 58                                          XXXX
```

Comment:
talk about a synthetic test case :p but it's nice to have until we find an actual sample in the wild

Reply:
tbf, I would hope we don't... it would be pretty broken. This was just a file I had made preemptively to appease the coverage bot.